duckdb-vortex/vcpkg.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": [
-    "openssl"
+    "openssl",
```

Comment:
Unrelated to this PR, I guess we can get rid of openssl?!

Reply:
LETS, bye bye